mbedtls/src/lib.rs

```diff
@@ -56,23 +59,23 @@ pub mod x509;
 mod private;
 
 // needs to be pub for global visiblity
-#[cfg(feature = "spin_threading")]
+#[cfg(any(feature = "spin_threading", feature = "rust_threading"))]
 #[doc(hidden)]
 pub mod threading;
 
 // needs to be pub for global visiblity
-#[cfg(all(feature = "std", not(target_os = "none")))]
+#[cfg(feature = "force_aesni_support")]
 #[doc(hidden)]
 #[no_mangle]
-pub unsafe extern "C" fn mbedtls_log(msg: *const std::os::raw::c_char) {
-    print!("{}", std::ffi::CStr::from_ptr(msg).to_string_lossy());
+pub extern "C" fn mbedtls_aesni_has_support(_what: u32) -> i32 {
+    return 1;
 }
 
 // needs to be pub for global visiblity
 #[cfg(feature = "force_aesni_support")]
 #[doc(hidden)]
 #[no_mangle]
-pub extern "C" fn mbedtls_aesni_has_support(_what: u32) -> i32 {
+pub extern "C" fn mbedtls_padlock_has_support(_what: u32) -> i32 {
```

Comment:
Don't add this

Reply:
Thanks Jethro. I have added the new functions as defines in config.rs.